src/lib/performance-monitor.ts

```
@@ -0,0 +1,47 @@
+  }
+
+  static logSummary(steps: Record<string, number>): void {
+    const total = Object.values(steps).reduce((sum, time) => sum + time, 0);
+    console.log('[Performance Summary]');
+    Object.entries(steps).forEach(([step, time]) => {
+      const percentage = ((time / total) * 100).toFixed(1);
+      console.log(`  ${step}: ${time}ms (${percentage}%)`);
+    });
+    console.log(`  Total: ${total}ms`);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)